README.md

```diff
@@ -361,7 +364,10 @@ spec:
   service:
     apiVersion: com.example/v1alpha1
     kind:       AccountService
-    name:       prod-account-service
+    selector:
```

Comment:
I wonder whether combining the Application label selector and Service label would be confusing to users.
How about renaming the title of the existing example to `Application Label Selector Example Resource` and then add another example with the title `Service Label Selector Example Resource`?

Reply:
That's a valid point - I opted for a single sample for brevity.  I don't feel too strong either way, but my vote would be to keep as proposed in the PR so that we have a single sample that covers both application & service.